lib/validate_configs.js

```diff
@@ -8,8 +8,8 @@ var getModule = require('./file_utils').getModule;
 function getConnectorSchema(name, configFile) {
     var opsDir = 'noOP';
 
-    if (configFile.teraslice && configFile.teraslice.teraslice_ops_directory) {
```

Comment:
Does this belong here? It references teraslice configuration from terafoundation.